adapters/MoPub/MoPubAdapter/MoPubAdapterMediatedNativeAd.m

```diff
@@ -43,8 +43,8 @@ - (instancetype)initWithMoPubNativeAd:(nonnull MPNativeAd *)moPubNativeAd
 
     CGFloat defaultImageScale = 1;
     if (downloadedImages != nil) {
```

Comment:
Do you have any guidance here? It's no-op if the check is removed, because `downloadedImages` will still be nil since it's sent from here (https://github.com/googleads/googleads-mobile-ios-mediation/blob/2ad71595a5ca3f77fe6071a220aa3eb38a2852d3/adapters/MoPub/MoPubAdapter/GADMAdapterMoPub.m#L212), and I don't see why the following code block doesn't download the main image (because it downloads the icon/logo): https://github.com/googleads/googleads-mobile-ios-mediation/blob/2ad71595a5ca3f77fe6071a220aa3eb38a2852d3/adapters/MoPub/MoPubAdapter/MoPubAdapterMediatedNativeAd.m#L49-L54

Reply:
Ah, I commented in the wrong file. I actually intended this comment for this line of code: https://github.com/googleads/googleads-mobile-ios-mediation/blob/a450119667514888607e4635dcd8f7ae41e40c32/adapters/MoPub/MoPubAdapter/GADMAdapterMoPub.m#L207

Right now, the adapter checks the `shouldDownlaodImages` to determine whether or not to download any images. Since the publisher expects the media view to be populated, the adapter should download the main image irrespective of the preference set for `GADNativeAdImageAdLoaderOptions'` `disableImageLoading` option.

Reply:
Thanks! I opened a new PR here and we can close this one: https://github.com/googleads/googleads-mobile-ios-mediation/pull/53.